Formula/a/aws-sdk-cpp.rb

```diff
@@ -1,13 +1,17 @@
 class AwsSdkCpp < Formula
   desc "AWS SDK for C++"
   homepage "https://github.com/aws/aws-sdk-cpp"
-  # aws-sdk-cpp should only be updated every 15 releases on multiples of 15
   url "https://github.com/aws/aws-sdk-cpp.git",
       tag:      "1.11.285",
       revision: "8213a588808acf5148b0f739e0b558879955310f"
   license "Apache-2.0"
   head "https://github.com/aws/aws-sdk-cpp.git", branch: "main"
 
+  livecheck do
+    url :stable
```

Comment:
```suggestion
```

https://github.com/Homebrew/brew/pull/16932 is merged, so we can omit `url` here now.